CONTRIBUTORS.md

```diff
@@ -0,0 +1,47 @@
+
+
+# How to contribute
+
+Mesmer is a project to extract and export akka specific metrics.
+
+Projects is comprised of 3 major parts:
+- java agent (called `agent` from now on) - a piece of technology allowing to modify classes during load.
+- akka extension (called `extension` from now on)  - this is where all custom metrics computation happens.
+- application that is being instrumented - in repo example, but this in theory this could be any application that uses akka.
+
+The most interesting parts are java agent and akka extension. They have to work in tandem - `agent` creates a foundation on which `extension` can build and calculate metrics.
+It's good to use an example to get a better understanding - `agent` allows to trigger an event whenever http request starts and completed 
+(we found places where those should be triggered by looking at akka source code and then modify those classes during load). 
+`extension` then catch those events and calculate duration between the two and save it in opentelemetry instrument. There will be more about opentelemetry role in the product later in this document.
+
+
+## Extension
+
+It's a part that is responsible for calculating and exporting metrics. The fact that it's akka extension is an implementation detail, but it's worth to know that.
+Akka extension are plugins that grant additional functionality to akka and can be enabled with configuration - this is the approach that we took.
```

Comment:
```suggestion
Akka extensions are plugins that grant additional functionality to Akka and can be enabled with configuration - this is the approach that we took.
```

It might make sense to replace this with a link to Akka extension docs for the users to get the general idea.